test/perf/dotnet-tools/BenchmarkDotNet.Extensions/ExclusionFilter.cs

```diff
@@ -0,0 +1,52 @@
+﻿using BenchmarkDotNet.Filters;
+using BenchmarkDotNet.Running;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BenchmarkDotNet.Extensions
+{
+    class ExclusionFilter : IFilter
```

Comment:
this is another example of something that we should add to BenchmarkDotNet so you could just use it out of the box without the need to copy anything

Reply:
Yeah, it could be generally useful.

To be honest, I didn't evaluate what would be needed by PowerShell and what would not when copying the `BenchmarkDotNet.Extensions` and `Reporting`. One consideration is that we probably will have to sync those sources with dotnet/performance from time to time, so the syncing would be less trivial if we selectively includes only part of them in PowerShell. But I also think it makes sense to only keep what we need in PowerShell repo, so we know we are actually using what's included from the dotnet/performance repo.